examples/fabric/v4/cloudRouterConnectivity/cloudRouter2serviceprofile/terraform.tfvars.example

```diff
@@ -0,0 +1,16 @@
+equinix_client_id      = "Equnix_Client_Id"
+equinix_client_secret  = "Equinix_Client_Secret"
+connection_name = "cloudRouter_gen_csp"
+connection_type = "IP_VC"
+notifications_type = "ALL"
+notifications_emails = ["example@equinix.com"]
+bandwidth = 50
+redundancy = "PRIMARY" || "SECONDARY"
```

Comment:
Sorry, this can't support both at the same time. Was looking for something like: `redundancy = "PRIMARY" # Can either be PRIMARY or SECONDARY`

Reply:
@srushti-patl one last requested change, but I'll approve so that you can make the change and move it on to the other approver.